apps/nextjs-website/scripts/fetch-docs.sh

```diff
@@ -16,6 +18,9 @@ rm -r ./docs
 # move files
 mv ./.tmp/devportal-docs-$hash*/docs .
 
+# Recursively find files named "README (1)" and rename them to "README_1"
+find ./docs -type f -name "README (1).md" -exec ./scripts/rename.sh {} + 
+
```

Comment:
I'm not sure about this additional step. It is needed because after the migration the system throw some error due to a path with spaces? Can it be solved on [src/lib/api.ts](https://github.com/pagopa/developer-portal/pull/240/files#diff-fab5d3c9dcc2b410005986fc807bd878f86848ba4d4dba55ad6408526bdaa9cf)?